<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TINPNQ-DWX-0102" failures="1" tests="3" name="CSMC" time="127.944" errors="0" timestamp="09 Jan 2020 06:40:27 GMT">
  <testcase classname="com.gcc.connector.Sanity.CSMC" name="csmcConnect" time="5.202"/>
  <testcase classname="com.gcc.connector.Sanity.CSMC" name="submissionCSMCstatus" time="122.523">
    <failure type="java.lang.AssertionError" message="expected [0] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [200]
at com.gcc.connector.Sanity.CSMC.submissionCSMCstatus(CSMC.java:249)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- submissionCSMCstatus -->
  <testcase classname="com.gcc.connector.Sanity.CSMC" name="sessionterminatesTC" time="0.0">
    <skipped/>
  </testcase> <!-- sessionterminatesTC -->
</testsuite> <!-- CSMC -->
